### ISSUE TITLE
How to handle "read only" attributes

### ISSUE THREAD

**Issue by bvandenbos** [chingor13/json_api_client#44]
Need some way to flag properties as readonly so they aren't posted to the server.

```ruby
class MyResource < JsonApiClient::Resource
   #...
   property :created_at, type: :time # <== this field is getting posted to the server but it's not allowed
end
```

**Comment by Fivell** [comment]
@bvandenbos , did you find workaround ?

**Comment by Fivell** [comment]
possible solution is to override 

```ruby
def serialized_attributes
    attributes.except(*RESERVED).except(protected_attributes)
end

def protected_attributes
   [:total_amount]
end
```
@chingor13 , what do you think to make this work out of the box?

**Comment by chingor13** [comment]
I'm thinking of implementing `serializable_hash` on the resource class. It's what `ActiveModel::Serializer` uses to define the data to be serialized.

**Comment by senid231** [comment]
also, i think we should have possibility to mark relationships as read only
as result they should not append to update requests

**Comment by senid231** [comment]
by the way, i think we should have different serialized_attributes for create and update

**Comment by chingor13** [comment]
should be addressed by #84
